praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
```

Comment:
While it breaks the preexisting convention, the `get_` prefix in front of many of the methods is something that should be deprecated, and removed in a future major version.

Could you please call call this method `default_subreddits`? 

Also, could you please sort the function lexicographically among the instance methods of the class?